src/develop/logic/exceptions/handling-mechanism.md

```diff
@@ -23,6 +23,12 @@ As an example, consider an Action B raising a User Exception named MyUserExcepti
 
 You should have, at least, one Exception Handler in your application flow to inform and allow the end user to continue to navigate.
 
+<div class="info" markdown="1">
+
+**OnApplicationReady** is a special callback that is not covered by the Global Exception Handler.
```

Comment:
I wouldn't call it "callback". Is this a common term in our documentation? In service studio it is called an event handler
.
And I would add in the end that it should have the error handling implemented in the action itself.